tests/io/loaders/test_general.py

```
@@ -0,0 +1,24 @@
+from readii.io.loaders.general import *
```

Ruff (0.8.2)

`from readii.io.loaders.general import *` used; unable to detect undefined names

(F403)